app/config-recommend/config-recommend.module.css

```
@@ -0,0 +1,455 @@
+
+.resultActions {
+  padding-top: 18px;
+  border-top: 1px solid var(--recommend-border);
+}
+
+.copyStatus {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)